### DISCUSSION TITLE
Stuck with 0.9.5.7 (NET 5.0) - what is the proper Chart.js version ?

### DISCUSSION THREAD

**Discussion by hannunkoneistus** [Megabit/Blazorise#4231] [General]
Hi,
Being stuck with Visual Studio 2019, I can't go higher than 0.9.5.7 while evaluating this framework. Using Chart.js 3.7.1 I get interop error "Cannot read properties of undefined (reading 'tooltips')"

What is the proper Javascript version? Thank you.

EDIT: With trial and error, I found out that 2.9.4 is the latest working Chart.js for 0.9.5.7

**Comment by stsrki** [comment]
Based on the 095 branch this is the last version https://github.com/Megabit/Blazorise/blob/rel-0.9.5/Demos/Blazorise.Demo.AntDesign/wwwroot/index.html#L46
